registration/templates/registration/registration_form.html

```diff
@@ -20,6 +20,9 @@
         {% include row with field=form.accept_terms only %}
         {% include row with field=form.subscribe_to_updates only %}
     {% endwith %}
+    {% with row="registration/hidden_field.html" %}
```

Comment:
@itsjeyd  I think you should just put `{{ field }}` directly in registration_form.html here. You don't need all that extra HTML to create a row and columns just to contain a hidden field.